node/src/main/scala/org/bitcoins/node/Node.scala

```diff
@@ -279,4 +296,28 @@ trait Node extends NodeApi with ChainQueryApi with P2PLogger {
   override def epochSecondToBlockHeight(time: Long): Future[Int] =
     chainApiFromDb().flatMap(_.epochSecondToBlockHeight(time))
 
+  case object ManagePeersRunnable extends Runnable {
```

Comment:
This needs to be in `NodeAppConfig`, else we are going to have resource related issues. See #2957